setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 [flake8]
 application-import-names = orderable
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D105,D203,D204
```

Comment:
Looking at the list of errors, I think there are only one or two that are particularly significant.  A lot of them are things we all do by habit, although they're still useful for newcomers in the future so that's not a good reason to remove it.  (There's also a couple of stupid ones like "First line should be in imperative mood" :P)